### DISCUSSION TITLE
Set android rustdesk client parameters

### DISCUSSION THREAD

**Discussion by tyaps** [rustdesk/rustdesk#6353] [Q&A]
On Android, you can send an intent to the application. For example, through adb I can start the rustdesk client with `am start -n com.carriez.flutter_hbb/.MainActivity`
Can I set a password or other parameters through an intent call, for example something like `am start -n com.carriez.flutter_hbb/.MainActivity -a com.carriez.flutter_hbb.setParams --es password 123 --es idserver 127.0.0.1 -- es key FwYY7KsJ5vk=`

**Comment by rustdesk** [comment]
How about do it with uni link? is it feasible?

Below is answered from chatgpt, not sure if it is working.
```
adb shell am start -a android.intent.action.VIEW -d "uni-link://your_uni_link_url"
```

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/issues/616#issuecomment-1837468064
